commit c0cb36603a182a6fb6943293101da8eea856ff65
Author: Damien Ciabrini <damien.ciabrini@gmail.com>
Date:   Thu Dec 24 16:40:42 2009 +0100

    xkeyboard-config: Base rules for Apple Aluminium Keyboards
    
    Additional base rules to define the geometry and symbols to use for ANSI, ISO and JIS keyboards
    
    Signed-off-by: Damien Ciabrini <damien.ciabrini@gmail.com>

diff --git a/rules/base.m_k.part b/rules/base.m_k.part
index 6d5eda0..f9ff97d 100644
--- a/rules/base.m_k.part
+++ b/rules/base.m_k.part
@@ -9,6 +9,7 @@
   evdev		=	evdev
  $nokiamodels	=	evdev
   olpc		=	evdev
+ $applealu	=	macintosh(alukbd)
   macintosh_hhk	=	macintosh(hhk)
   macintosh_old	=	macintosh(old)
  $macs   	=	macintosh
diff --git a/rules/base.m_s.part b/rules/base.m_s.part
index a032e0a..0f1bcce 100644
--- a/rules/base.m_s.part
+++ b/rules/base.m_s.part
@@ -48,4 +48,5 @@
   vsonku306     =       +inet(microsoftprooem)
   $inetkbds     =       +inet(%m)
   $maclaptop    =       +inet(apple)+level3(enter_switch)
+  $applealu     =       +inet(apple)
   $macs	        =       +inet(apple)
